.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+_Replace this content with your own_
+
+## Checklists
+
+### About the changes
+
+- [ ] Tests added if necessary
+- [ ] man-pages (`./man`) updated if necessary
+- [ ] Formatted properly (e.g. Restyled passes)
+
+### About the PR
+
+- [ ] Descriptive, imperative-tense title
+- [ ] Body explaining the _why_ of the change
+- [ ] `breaking-change` or `enhancement` label applied if appropriate
```

Comment:
The new release action can build automatic release-notes that are quite similar to our own action, so I think we should move to it. It uses PR titles and I configured it to use these labels to group things.